### PR TITLE
Docs dev setup

### LINKED ISSUE #198: Create developer setup guide
towards acheiving #187 

> Step-by-step guide for a developer on how to setup an environment in which they can run, test and work on the project

### PR DESCRIPTION
**Pull request by eliascodes**
Fixes #198 
Related to #187 

### Main changes
I've started by adding a simple dev setup guide. I anticipate it (and in particular the config guide) will change significantly over the course of this phase, but it's good to have this to start with.

I also added all contributors to the `package.json`. Didn't know everyone's e-mail but can add them if you let me know what they are.

### Misc
See my commits for a simple example of the kind of thing I was talking about in #188, in particular the multi-line messages, the fact that all changes in a commit are related, etc.

@astroash I'm sure you'll be happy to notice I'm using emoji. I'm using [gitmoji](https://kapeli.com/cheat_sheets/Gitmoji.docset/Contents/Resources/Documents/index) as the convention for emoji prefixes, not just prepending random ones. This helps quickly visually assess what a commit is doing.

Also I'm not using GitHub's colon-enclosed shortcuts, I'm using the macOS emoji keyboard, that means the emoji will appear everywhere that supports emoji (e.g. the terminal), not just in the GitHub UI.

@RogeredBacon I don't know how you feel about this whole emoji thing, it's up to you, as long as your commit messages are moving in the direction of #188 I'm happy 😄

### CODE REVIEW

**Comment by astroash** [comment]
> I'm using gitmoji as the convention for emoji prefixes, not just prepending random ones.

@eliascodes I'm not going to lie... but I'm probably not going to do this...

**Comment by RogeredBacon** [comment]
I don't mind one way or the other, if you both you guys are doing it I'll try to remember

File: docs/config_guide.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Configuration Guide
+Currently all configuration is done via a single git-ignored configuration file in the project root.
+
+To create a project configuration, create a file named `config.env` in the project root. Populate it with `=`-separated key-value pairs. Ensure at least the following parameters are provided:
+
+```sh
+# Used to sign JWT for community business (CB) users
```
Comment (eliascodes) [inline docs/config_guide.md]: @astroash @RogeredBacon this was missed in the review. `SECRET` is used to key the HMAC operation when storing passwords. See #213

File: docs/config_guide.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+SECRET=...
+
+# Used to sign JWT for admin users
+ADMIN SECRET=...
+
+# what?
+POSTMARK_SERVER=...
+
+# what?
```
Comment (eliascodes) [inline docs/config_guide.md]: Also missed in the review.

**Pull request merged**